src/services/apiService.js

```diff
@@ -332,13 +333,28 @@ angular.module('openshiftCommonServices')
     });
   };
 
+  console.log('available kinds:');
+  console.log('available kinds:');
+  console.log('available kinds:');
+  console.log('available kinds:');
+  console.log(calculateAvailableKinds(true));
+  console.log(JSON.stringify(calculateAvailableKinds(true)))
```

Comment:
Remove log messages

Reply:
OOp, sorry, should have said "Do not review" on the last one, just pushed it to switch gears back to drawer.  🙁 

This will get cleaned up in my next pass.